Competition2018/src/main/java/competition/subsystems/offboard/OffboardInterfaceSubsystem.java

```diff
@@ -83,13 +87,37 @@ public void sendSetCurrentCommand(int commandId) {
         return queueContents;
     }
     
+    private void sendScoringPlacement() {
+        //Sequence - Near, Scale, Far
+        //Left - 1, Right - 0
+        ArrayList<Integer> sequence = new ArrayList<Integer>();
+        for(MatchData.GameFeature feature: MatchData.GameFeature.values()) {
+            if(MatchData.getOwnedSide(feature).equals(MatchData.OwnedSide.LEFT)) {
+                sequence.add(0);
+            }
+            else if(MatchData.getOwnedSide(feature).equals(MatchData.OwnedSide.RIGHT)) {
+                sequence.add(1);
+            }
+            else {
+                //Insert Error Message as it is unknown
+            }
+        }
+        int total = 0;
+        for (Integer i : sequence) { 
+            total = 10*total + i;
```

Comment:
I would suggest a slightly different approach. It's the same concept, but works a bit better with the binary internal representation. You can use a "bit shift" `>>` to store each value as a single binary bit. Additionally, I'd suggest moving this logic into the "pack" method. That method should take the side info and figure out how to do the bit manipulation.

Reply:
Thanks!